src/main/java/eu/europeana/metis/sandbox/domain/DatasetMetadata.java

```diff
@@ -0,0 +1,170 @@
+package eu.europeana.metis.sandbox.domain;
+
+import eu.europeana.metis.sandbox.common.locale.Country;
+import eu.europeana.metis.sandbox.common.locale.Language;
+
+/**
+ * DatasetMetadata class used in {@link eu.europeana.metis.sandbox.service.dataset.DatasetGeneratorService} to provide dataset
+ * metadata to generate it.
+ */
+public class DatasetMetadata {
+  private String datasetId;
+  private String datasetName;
+  private Country country;
+  private Language language;
+
+  private DatasetMetadata(DatasetMetadataBuilder datasetMetadataBuilder) {
+    setDatasetId(datasetMetadataBuilder.datasetId);
+    setDatasetName(datasetMetadataBuilder.datasetName);
+    setCountry(datasetMetadataBuilder.country);
+    setLanguage(datasetMetadataBuilder.language);
+  }
+
+  /**
+   * New builder builder.
+   *
+   * @return the builder
+   */
+  public static DatasetMetadataBuilder Builder() {
```

Comment:
Builder should start lower cased, it's a method .

Reply:
yes. correct. renamed.